python/build_images.py

```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/python3.6
+
+import subprocess
+
+string=subprocess.check_output(["find", ".", "-name", "*.drawio", "-print"])
+arr=string.splitlines()
+for s in arr:
+    sD=s.decode("utf-8")
+    subprocess.run(["drawio-batch", sD, 'source/images/'  + sD[22:-7] + ".png"])
```

Comment:
Python 3.6 has [Pathlib](https://docs.python.org/3.6/library/pathlib.html) built in. You should use that for any file path manipulation when working in Python instead of using string indexing, there is a backport for Python2 as well. Then to get the name of a file it becomes just `Path(sD).name` or `Path(sD).stem` if all you want is the file name without its extension.